blog/2022-04-06-multi-cluster-management.md

```diff
@@ -0,0 +1,285 @@
+            - type: webservice
+              traits:
+              - type: scaler
+                properties:
+                  replicas: 5
+```
+At the same time, we hope that only AWS clusters can get high availability. Then we can expect KubeVela's workflow give us a hand. We use the following workflow: it aims to deploy this app by, first distributing to Beijing's clusters through the `deploy-beijing` policy, then distributing 5 copies to clusters which were labeled as AWS:
+```
+...                
+                properties:
+                  replicas: 5
+  workflow:
+    steps:
+      - type: deploy
+        name: deploy-beijing
+        properties:
+          policies: ["beijing-clusters"]
+      - type: deploy
+        name: deploy-aws
+        properties:
+          policies: ["override-high-availability","topology-aws"]
+```
+Then we attach the above policy and workflow to `intermediate.yaml` and make it to `advanced.yaml`:
+```
+...
+  policies:
+    - type: topology
+      name: beijing-clusters
+      properties:
+        clusters: ["beijing-1","beijing-2"]
+    - type: topology
+      name: topology-aws
+      properties:
+        clusterLabelSelector:
+          provider: AWS
+    -  type: override
+       name: override-high-availability
+       properties:
+          components:
+            - type: webservice
+              traits:
+              - type: scaler
+                properties:
+                  replicas: 5
+  workflow:
+    steps:
+      - type: deploy
+        name: deploy-beijing
+        properties:
+          policies: ["beijing-clusters"]
+      - type: deploy
+        name: deploy-aws
+        properties:
+          policies: ["override-high-availability","topology-aws"]
+```
+Then deploy it, view the status of the application:
+```
+➜   vela up -f advanced.yaml
+Applying an application in vela K8s object format...
+"patching object" name="example-app" resource="core.oam.dev/v1beta1, Kind=Application"
+✅ App has been deployed 🚀🚀🚀
+    Port forward: vela port-forward example-app
+             SSH: vela exec example-app
+         Logging: vela logs example-app
+      App status: vela status example-app
+  Service status: vela status example-app --svc hello-world-serverapplication.core.oam.dev/podinfo-app configured
+  
+➜   vela status example-app
+
+...
+
+  - Name: hello-world-server
+    Cluster: us-west-1  Namespace: default
+    Type: webservice
+    Healthy Ready:5/5
+    Traits:
+      ✅ scaler      ✅ gateway: Visiting URL: testsvc-mc.example.com, IP: 192.168.40.10
+
+```
+
+The above all are what we'd like to share with you for this time, thank you for reading and trying them out.
+
+[We invite you to explore KubeVela v1.3 for more] (https://KubeVela.net/en/docs/install) to meet further complex requirements on business, such as dig deep in differentiated configurations to use `override` application policy to either override all resources on one type or only certain specific components.
```

Comment:
we can add a link of the dig deep to our multi-cluster chapter, after #571 is accepted.

Reply:
hold up for https://github.com/oam-dev/kubevela.io/pull/571